bad_apps_blog/blog.py

```diff
@@ -75,13 +79,15 @@ def update(id):
         if error is not None:
             flash(error)
         else:
+            current_app.logger.info(f'post {id} updated')
```

Comment:
This line may also introduce a log injection vulnerability via the 'id' parameter